utkdigitalinitiatives/utk-fedora-fedoragsearch-solr-config
# Address DIT-1278.

**Jira Issue**: [DIT-1278](https://jirautk.atlassian.net/browse/DIT-1278)

## What does this Pull Request do?

Removes all autogenerated implicit fields form the MODS request.

## What's new?

I deleted all the slurp_mods transforms from this so that the only fields created are explicit.

## How should this be tested?

A description of what steps someone could take to:

1. Add an object to vagrant
2. Look at its Solr document:  [http://localhost:8080/solr/collection1/select?q=PID%3A%22test:1%22&fl=*&wt=json&indent=true](http://localhost:8080/solr/collection1/select?q=PID%3A%22test:1%22&fl=*&wt=json&indent=true)
3. Overwrite the transform here.: /vhosts/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/slurp_all_MODS_to_solr.xslt
4. Restart fedoragsearch (stop then start): http://localhost:8080/manager/
5. Update gsearch for your object with curl or python or GUI:
6. Look at the Solr document again.  It should only have MODS fields that start with utk_.

```python

import requests

my_pid = 'test:1'
requests.post(f'http://localhost:8080/fedorafedoragsearch/rest?operation=updateIndex&action=fromPid&value={my_pid}', auth=('fedoraAdmin', 'fedoraAdmin'))

```

## Interested parties

@DonRichards @CanOfBees @mlhale7 @pc37utn
